src/openrct2/platform/shared.c

```diff
@@ -761,7 +775,23 @@ void platform_set_cursor_position(sint32 x, sint32 y)
 
 uint32 platform_get_ticks()
 {
-	return SDL_GetTicks();
+#ifdef _WIN32
```

Comment:
Shouldn't these be in the seperate platform files? Not really `shared` if there are so much platform checks